acceptance/acceptance_test.go

```diff
@@ -43,6 +43,9 @@ import (
 const (
 	runImage   = "pack-test/run"
 	buildImage = "pack-test/build"
+	version    = "1.0.2"
+	Type       = "project"
+	metadata   = "https://github.com/buildpacks/pack"
```

Comment:
```suggestion
```

It doesn't look like there's much value in these being package constants since they only get used in one location. I would suggest you inline them. See suggestion below.